frontend/src/Editor/Box.jsx

```diff
@@ -15,6 +15,8 @@ import { Multiselect } from './Components/Multiselect';
 import { Modal } from './Components/Modal';
 import { Chart } from './Components/Chart';
 import { Map } from './Components/Map';
+import OverlayTrigger from 'react-bootstrap/esm/OverlayTrigger';
```

Comment:
Just curious, what is the difference between this line and `import OverlayTrigger from 'react-bootstrap/OverlayTrigger';`?

Reply:
Just noticing this as it was auto-imported by VS Code. Both are the same, the only difference being one is a CJS module (Common JS) and the other is an EcmaScript module (ESM). 
The react-bootstrap library is using multiple formats for distribution.

Reply:
Refactoring the import now to keep it consistent with other places where <OverlayTrigger /> has been imported.